mkdata.py

```diff
@@ -9,27 +10,54 @@
 Licensed under the BSD 3-clause license.
 """
 
+try:
+    import click
+except ImportError:
+    from mock import Mock
+    click = Mock()
+    def call(*args, **kargs):
+        def decorator(f):
+            return f
+        return decorator
+    click.command = call
+    click.argument = call
+    click.option = call
```

Comment:
i was hoping to keep this backward-compatible to non-click environment, so that this can run without virtualenv and all...